### PR TITLE
testing: Allow to override service account name and secret name in STRICT security mode

### PR DESCRIPTION
**Pull request by mhrabovcin**
https://jira.mesosphere.com/browse/DCOS-18083

### CODE REVIEW

File: frameworks/helloworld/tests/test_tls.py [hunk 1]
```diff
@@ -37,16 +37,7 @@
 
 
 @pytest.fixture(scope='module')
-def dcos_security_cli():
```
Comment (benclarkwood) [inline frameworks/helloworld/tests/test_tls.py]: @mhrabovcin why drop this?

**Comment by benclarkwood** [comment]
Should this fix all the TLS tests in strict?

File: frameworks/helloworld/tests/test_tls.py (hunk 1 above)
Comment (mhrabovcin) [inline frameworks/helloworld/tests/test_tls.py]: We're dropping this becuase `sdk_security.create_service_account` does the same thing so its a dupe.

**Comment by mhrabovcin** [comment]
@benclarkwood Yes, I've tested both helloworld and kafka in strict security mode, both tests passed.

**Pull request merged**